jscomp/frontend/ast_external_process.ml

```diff
@@ -449,11 +475,16 @@ let process_obj (loc : Location.t) (st : external_desc) (prim_name : string)
               | Extern_unit -> assert false
               | Poly_var _ ->
                 Location.raise_errorf ~loc
-                  "%@obj label %s does not support such arg type" name
+                  "%@obj label %s does not support such arg type" label
```

Comment:
Separate from this PR, but these errors would be good to clean up too at some point.